### ISSUE TITLE
adapt for mySQL

### ISSUE THREAD

**Issue by Mad-Dad** [santiq/bulletproof-nodejs#98]
I have been dealing with javascript for many years and realize it's more powerful than whats used for standard web stuff.  but this is over my head.  I really like the concepts but need to understand it better.
So my question is would it be difficult to convert this to mysql, a locally installed instance?

**Comment by klvenky** [comment]
I don't think so @Mad-Dad. It'll be easy to replace the database initiation step using any standard database layer. I'll try to create a sample for that.

**Comment by dheeraj-rn** [comment]
@Mad-Dad I've made a sample project with Sequelize and PostgreSQL. It should be same for MySQL if you are using sequelize. You can checkout the code here: [https://github.com/dheeraj-rn/nse](https://github.com/dheeraj-rn/nse). I used js instead of ts.
